config/config.go

```
@@ -0,0 +1,65 @@
+}
+type ChatGptConfig struct {
+	Sdk string `yaml:"sdk"`
+}
+type QQConfig struct {
+	Email string `yaml:"email"`
+	Key   string `yaml:"key"`
+}
+
+func NewDatabaseConfig(vs *ViperSetting) *DatabaseConfig {
+	var databaseConfig = &DatabaseConfig{}
+	vs.ReadSection("database", &databaseConfig)
```

golangci-lint (1.62.2)

Error return value of `vs.ReadSection` is not checked

(errcheck)